aws-synthetics-canary/src/main/java/com/amazon/synthetics/canary/CreateHandler.java

```diff
@@ -159,7 +168,7 @@
         try {
             proxy.injectCredentialsAndInvokeV2(createCanaryRequest, syntheticsClient::createCanary);
         } catch (final ValidationException e) {
-            throw new CfnInvalidRequestException(e.getMessage());
+            throw new CfnAlreadyExistsException(ResourceModel.TYPE_NAME, model.getPrimaryIdentifier().toString());
```

Comment:
How do you know that a validation exception is always because it already exists? Shouldn't we check to make sure the `ValidationException` is *actually* a duplicate error?